src/common/plugin/types.ts

```diff
@@ -17,7 +17,8 @@ export enum NetworkEnum {
   Testnet = 'testnet'
 }
 
-export type CurrencyFormat = 'bip32' | 'bip44' | 'bip49' | 'bip84'
+export type CurrencyFormat = ReturnType<typeof asCurrencyFormat>
+export const asCurrencyFormat = asValue('bip32', 'bip44', 'bip49', 'bip84')
```

Comment:
Discover if we can remove `bip84`.

Reply:
I don't think we can safely remove this from the union type because this type appears to be a general purpose union of all formats.